backend/app/store/service/service.go

```diff
@@ -725,6 +740,12 @@ func (s *DataStore) SetMetas(siteID string, umetas []UserMetaData, pmetas []Post
 		if um.Verified {
 			errs = multierror.Append(errs, s.SetVerified(siteID, um.ID, true))
 		}
+		// this code doesn't delete user details in case they are not set in import but present in DB already
+		if um.Details != (engine.UserDetailEntry{}) && um.Details.Email != "" {
```

Comment:
(engine.UserDetailEntry{}) ? why () here?

Reply:
Checking against straight `engine.UserDetailEntry{}` produces syntax error, it surprised me as well. Apparently, that's the way of checking against empty (but not nil) structure. [Here is what I found on it now](https://stackoverflow.com/a/28447372/961092).

Reply:
hmm, looks ugly to me. Maybe just check `um.Details.UserID != ""`  ?